src/pages/Home/Description.js

```diff
@@ -143,7 +141,7 @@ export default function Description(){
         <Information>
             <P {...(isMobile && {onClick:() => setCollapsed(!collapsed)})}>
                 <span>{t('information.title')}</span>
-                <Arrow collapsed={collapsed} />
+                <Arrow collapsed={collapsed}>&#x2BC6;</Arrow>
```

Comment:
I think we need a real icon here, the emoticion was not displayed on my side :/

Reply:
For me it's ok. I'll change for an icon